ext/ftp/tests/ftp_rawlist_basic1.phpt

```diff
@@ -1,7 +1,5 @@
 --TEST--
 Testing ftp_rawlist basic functionality
---CREDITS--
```

Comment:
I dont know if doing this was right honeslty ... I was told that --CREDITS-- was not suppose to be use anymore

Reply:
We usually do not remove already existing CREDITS, but in this case it might be okay. @carusogabriel, or do you object?

Reply:
> We usually do not remove already existing CREDITS, but in this case it might be okay. @carusogabriel, or do you object?

That is what I thought but I was not sure, I will add it again, is ethically correct to keep them